### DISCUSSION TITLE
Welcome to flopr Discussions!

### DISCUSSION THREAD

**Discussion by ajfedorec** [ucl-cssb/flopr#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by silviabrochet** [comment]
Hello,

I have a file resulting from a reading with Tecan. It's not quite like the examples files that you provide and therefore the parsing doesn't work. Could you help with this?

Thanks a lot,

Silvia

[17062022_testplate2_timecourseR.csv](https://github.com/ucl-cssb/flopr/files/8926833/17062022_testplate2_timecourseR.csv)
[plate_layout.csv](https://github.com/ucl-cssb/flopr/files/8926834/plate_layout.csv)

**Reply by ajfedorec** [reply]
Hi, 
Thanks for the files. I'll write a parser in the next few days. What Tecan plate reader is this from?
Thanks,
Alex

**Reply by silviabrochet** [reply]
Hello, thanks a lot for doing this! 
The plate reader is the Infinite 200 PRO with the option Infinite M plex. 
Best,

Silvia

**Reply by ajfedorec** [reply]
Hi Silvia,

I have just pushed some changes to enable you to parse the file using the function infinite_parse(). You will need to update flopr by running devtools::install_github('ucl-cssb/flopr').

If you want to use calibration as well you will need to send me an example file with data from a single measurement rather than a timeseries as these are often saved in a different format.

Please let me know if this works for you.
Alex
